### PR TITLE
copy edits.  Plus fixed deprecated backend cudaq.set_target("nvidia-mqpu") 

### PR DESCRIPTION
**Pull request by mmvandieren**
cudaq.set_target("nvidia-mqpu") to 
cudaq.set_target("nvidia", option="mqpu")

### CODE REVIEW

File: docs/sphinx/applications/python/hadamard_test.ipynb [hunk 1]
```diff
@@ -354,7 +354,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The four matrix elements are shown below and now be classically processed to produce the eigenvalues."
+    "The four matrix elements are shown below and can now be classically processed to produce the eigenvalues."
```
Comment (schweitzpgi) [inline docs/sphinx/applications/python/hadamard_test.ipynb:357]:
```suggestion
    "The four matrix elements are shown below and can be classically processed to produce the eigenvalues."
```

File: docs/sphinx/applications/python/krylov.ipynb [hunk 2]
```diff
@@ -7,39 +7,45 @@
...
     "\n",
-    "$$ \\ket{\\Psi} = \\sum_{\\alpha} c_{\\alpha}\\ket{\\psi_{\\alpha}} = \\sum_{I=0}^d \\sum_{n=0}^s c_I^{(n)}\\hat{U}_n\\ket{\\Phi_I}  $$\n",
+    "$$ \\ket{\\Psi} = \\sum_{\\alpha} c_{\\alpha}\\ket{\\psi_{\\alpha}} = \\sum_{I=0}^d \\sum_{n=0}^s c_I^{(n)}\\hat{U}_n\\ket{\\Phi_I}.  $$\n",
     "\n",
     "The energy of this state can be obtained by solving the generalized eigenvalue problem\n",
-    "$$ \\boldsymbol{Hc}=\\boldsymbol{Sc}E $$\n",
+    "$$ \\boldsymbol{Hc}=\\boldsymbol{Sc}E, $$\n",
     "\n",
-    "Where the elements of the overlap and Hamiltonian matrix are\n",
+    "where the elements of the overlap  are\n",
```
Comment (schweitzpgi) [inline docs/sphinx/applications/python/krylov.ipynb:32]: Extra space?

File: docs/sphinx/applications/python/krylov.ipynb [hunk 3]
```diff
@@ -7,39 +7,45 @@
...
     "\n",
-    "These matrix elements are computed using a quantum computer and the Hadamard test with a circuit shown below for the case of the overlap matrix elements (The Hamiltonian matrix elements circuit would include controlled application of the Hamiltonian in the circuit).  Once the matrices are constructed, the diagonalization is performed classically to produce an estimate for the ground state in question.\n",
+    "$$H_{\\alpha \\beta} = \\braket{\\psi_{\\alpha}|\\hat{H}|\\psi_{\\beta}} =   \\braket{\\Phi_I|\\hat{U}_m^{\\dagger}\\hat{H}\\hat{U}_n|\\Phi_J}.$$\n",
+    "\n",
+    "The matrix elements for $S$ are computed with the Hadamard test with a circuit shown below for the case of the overlap matrix elements. \n",
     "\n",
     "![Htest](./images/krylovcircuit.png)\n",
     "\n",
-    "The $2\\sigma_+$ term refers to measurement of the expectation value of this circuit with the $X+iY$ operator. \n"
+    "The $2\\sigma_+$ term refers to measurement of the expectation value of this circuit with the $X+iY$ operator. \n",
```
Comment (schweitzpgi) [inline docs/sphinx/applications/python/krylov.ipynb:44]:
```suggestion
    "The $2\\sigma_+$ term refers to measurement of the expectation value of this circuit with the $X+iY$ operator.\n",
```

File: docs/sphinx/applications/python/krylov.ipynb [hunk 4]
```diff
@@ -7,39 +7,45 @@
...
+    "\n",
+    "The Hamiltonian matrix elements are computed with a circuit that includes controlled application of the Hamiltonian.  Once the $H$ and $S$ matrices are constructed, the diagonalization is performed classically to produce an estimate for the ground state in question.\n",
```
Comment (schweitzpgi) [inline docs/sphinx/applications/python/krylov.ipynb:46]:
```suggestion
    "The Hamiltonian matrix elements are computed with a circuit that includes controlled application of the Hamiltonian. Once the $H$ and $S$ matrices are constructed, the diagonalization is performed classically to produce an estimate for the ground state in question.\n",
```

Comment (schweitzpgi) [reply docs/sphinx/applications/python/krylov.ipynb:46]: The two spaces after a period is kind of IBM Selectric style these days, but it looks like this document is using it everywhere, so probably stick with it here.

**Review by schweitzpgi — COMMENTED** [review]
Looks great. Thanks for the doc updates.

File: docs/sphinx/applications/python/krylov.ipynb (hunk 2 above)
Comment (mmvandieren) [reply docs/sphinx/applications/python/krylov.ipynb:32]: ah yes, it appears there is an extra space there.

Comment (schweitzpgi) [inline docs/sphinx/applications/python/krylov.ipynb:32]:
```suggestion
    "where the elements of the overlap are\n",
```

**Review by schweitzpgi — APPROVED** [review]

**Comment by schweitzpgi** [comment]
@mmvandieren This one needs the DCO commit signing as well.

File: docs/sphinx/applications/python/hadamard_test.ipynb [hunk 5]
```diff
@@ -26,18 +26,18 @@
...
-    "Which corresponds to the following circuit\n",
+    "we can combine the state preparation steps into the operator resulting in\n",
+    "$$\\bra{\\psi}O\\ket{\\phi} = \\bra{0}U_\\psi^\\dagger O U_\\phi\\ket{0},$$\n",
+    "which corresponds to the following circuit.\n",
     "![Htest2](./images/htestfactored.png)\n",
     "\n",
-    "By preparing this circuit, and repeatedly measuring the ancilla qubit, allows for estimation of the expectation value as $$P(0)-P(1) = Re \\bra{\\psi} O \\ket{\\phi}$$\n",
-    "\n",
+    "By preparing this circuit, and repeatedly measuring the ancilla qubit, we estimate the expectation value as $$P(0)-P(1) = Re \\bra{\\psi} O \\ket{\\phi}.$$\n",
+    "\,
```
Comment (khalatepradnya) [inline docs/sphinx/applications/python/hadamard_test.ipynb:40]: This made the notebook invalid.

**Pull request merged**